packages/shared/src/badukWithAbstractBoard/abstractBoard/PolygonalBoardHelper.ts

```diff
@@ -316,11 +316,13 @@ class PolygonalTile {
 }
 
 export function CreatePolygonalBoard(size: number) : Intersection[] {
+    
     const StartTile: PolygonalTile = new PolygonalTile(new Vector2D(0, 0));
     const Tiles: PolygonalTile[] = [StartTile];
     let tileQueue: PolygonalTile[] = [StartTile];
 
-    for (let i = 1; i < (size + 1) / 2; i++) {
+    for (let i = 1; i <= (size - 1) / 2; i++) {
```

Comment:
Is this a breaking change? Will this affect already-created polygonal boards?

Reply:
Good point, yes I believe this would potentially break existing games with polygonal boards.

The "bug" this change tries to fix is not really a problem actually. I just think that the size of the polygonal board grows more naturally with this.

Reply:
Okay good to know, breaking changes are okay - we don't have users to be bothered, just wanted to check!